demo/index.html

```diff
@@ -25,7 +25,7 @@ <h1 tabindex="0">focus-trap demo</h1>
     <div id="demo-default">
       <h2 id="default-heading">default behavior</h2>
       <p>
-        <button id="activate-default" aria-label="activate trap for 'defaults' demo">
+        <button id="activate-default" title="activate trap for 'defaults' demo">
```

Comment:
I changed `aria-label` to `title` because aria-label will completely override the visible label which is an Accessibility fail against [WCAG criteria 2.5.3: Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name)

Reply:
Still wondering about this. To reduce the changes in the PR that aren't directly related to adding the new feature you're working on, I'd like to revert this change from `arial-label` to `title`, even if it may be technically correct. My aim is not to provide a fully-accessible demo anyway, and as I pointed out in my previous comment on this thread, the `aria-label` already repeats the content of the button label in all cases, so there's no information being lost to someone potentially using a screen reader to try to navigate the demo page (and I don't think they would be successful today _anyway_, in spite of the `aria-label` or `title`).

Reply:
I've had an idea about this. 

What if I do this:
`<button aria-describedby="id-of-heading">Activate trap</button>`

Now for the default example a screen reader will read out
"button activate trap default behavior"

- It doesn't give off a tooltip
- it doesn't fail the WCAG criteria 
- it gives screen readers the extra context about what trap is being activated
- it teaches developers good accessibility practices

Reply:
In a separate PR 🙂 

But the `aria-label` here will also be read out, so wouldn't that result in "button activate trap for 'defaults' demo default behavior"?

Or you're suggesting _removing_ the `aria-label` because the button already has "activate trap" as a label, and now we'll point to the heading as the description?

We could also point to it with `aria-labelledby` in the same way (if we remove `aria-label`).

But again, in a separate PR. We've already added all the `add/removeClass()` stuff to this PR, which also has nothing to do with what this PR is really about... Technically, that stuff, and this label stuff, should go in a separate PR about "best practices and better accessibility for the demos" or something like that. Even 2 PRs.

Reply:
The label of the button is "activate trap". We don't want aria messing around with that in any way whatsoever. 

So no to aria-labelledby since that will make screen readers announce the button as "button default behaviour" instead of "button activate trap" 

Yes to removing aria-label in favour of aria-describedby.

I think I'll open up some smaller PRs to get the demo site improvements into master then we can work more on this feature.

Reply:
I've created a separate PR now to cover the demo site improvements

https://github.com/focus-trap/focus-trap/pull/411

Reply:
Wonderful, thank you!